src/shogun/multiclass/LaRank.h

```diff
@@ -378,6 +384,26 @@ namespace shogun
 			 * @return tau
 			 */
 			float64_t get_tau() { return tau; };
+			/** set upper limit for iterations in training
```

Comment:
newline before this.

Also, please start sentences with Captial letters -- proper English